packages/shared_preferences/shared_preferences_android/pigeons/messages.dart

```diff
@@ -37,7 +37,13 @@ abstract class SharedPreferencesApi {
 
   /// Adds property to shared preferences data set of type List<String>.
   @TaskQueue(type: TaskQueueType.serialBackgroundThread)
-  bool setStringList(String key, List<String> value);
+  bool setStringList(String key, String value);
```

Comment:
Let's at least call this `setPreEncodedStringList` or something, so it's clear why it's called string list but takes a `String`.